index.html

```diff
@@ -319,8 +277,7 @@
         Reacting to visibility changes
       </h2>
       <p>
-        The <a>task source</a> for these <a>tasks</a> is the <a>animation task
-        source</a>.
+        The <a>task source</a> for these <a>tasks</a> is the <a href="https://html.spec.whatwg.org/#user-interaction-task-source">user interaction task source</a>.
```

Comment:
@annevk @bzbarsky does this look reasonable? We don't have animation task source in HTML spec.. and visibility changes are typically due to user input (switching tabs, etc). Alternatively, should we define own task source for Page Visibility?